src/components/agent-inbox/components/settings-popover.tsx

```diff
@@ -106,6 +146,29 @@ export function SettingsPopover() {
                 onChange={handleChangeLangChainApiKey}
               />
             </div>
+            {!backfillCompleted && (
+              <div className="flex flex-col items-start gap-2 w-full border-t pt-4">
+                <div className="flex flex-col gap-1 w-full items-start">
+                  <Label>Update Inbox IDs</Label>
+                  <p className="text-xs text-muted-foreground">
+                    Update your inbox IDs to the new format that supports
+                    sharing links across machines.
+                  </p>
+                </div>
+                <Button
+                  onClick={handleRunBackfill}
+                  disabled={isRunningBackfill}
+                  className="flex items-center gap-2"
+                >
+                  <RefreshCw
+                    className={
+                      isRunningBackfill ? "animate-spin h-4 w-4" : "h-4 w-4"
+                    }
```

Comment:
I think we need to wrap this in `cn`
```suggestion
                    className={cn(
                      isRunningBackfill ? "animate-spin h-4 w-4" : "h-4 w-4"
                    )}
```

Reply:
will need to import too*